src/settings/OrderTemplates/OrderTemplateView/OrderTemplateView.js

```diff
@@ -24,12 +24,16 @@ import {
   Row,
 } from '@folio/stripes/components';
 
+import { ViewCustomFieldsRecord } from '@folio/stripes/smart-components';
+
```

Comment:
You can remove it. The whole idea is a create a logical group for imports.
// Extrenal

// Folio

// Internal